src/app/common/hooks/account/use-account-names.ts

```diff
@@ -24,9 +24,11 @@ export function useCurrentAccountDisplayName() {
 }
 
 export function useAccountDisplayName({ address, index }: { index: number; address: string }) {
-  const { data: names = [] } = useGetAccountNamesByAddressQuery(address);
+  const { data: names = [], isLoading } = useGetAccountNamesByAddressQuery(address);
   return useMemo(() => {
+    // TO-DO - add proper UI loader
+    if (isLoading) return 'Loading bns name...';
```

Comment:
```suggestion
    if (isLoading) return 'Loading bns name…';
```

unicode ellipsis char 🙃